lib/PEXv1.ts

```diff
@@ -140,6 +145,8 @@ export class PEXv1 {
    * @return the validation results to reveal what is acceptable/unacceptable about the passed object to be considered a valid presentation definition
    */
   public validateDefinition(presentationDefinitionV1: PresentationDefinitionV1): Validated {
+    JsonPathUtils.changePropertyNameRecursively(presentationDefinitionV1, '_const', 'const');
```

Comment:
Be aware that this method called 'validate' is changing its input argument!

Reply:
moved to SSITypeBuilder